Update dependencies to work with Vite and NextJS, combine css files into a single css file (#1510)
kedro-org/kedro-viz: RELEASE.md

- Include the ID number for the related PR (or PRs) in parentheses
-->

# Release 6.5.1

## Bug fixes and other changes

> - Updated dependencies to ensure compatibility with Vite and Next.js environments, combine css into a signle file for npm library (#1510)

Reviewer: ```suggestion
- Updated dependencies to ensure compatibility with Vite and Next.js environments; combine CSS into a single file when used as a React component. (#1510)
```

- Fix to search for a '<lambda' Python function in the sidebar. (#1497)
- Add favicon to Kedro-Viz. (#1509)
- Remove python upper-bound requirements and add KedroVizPythonVersion warning. (#1506)

> - Resolved various bugs affecting server-side rendering, updated dependencies to ensure compatibility with Vite and Next.js environments (#1508)

Reviewer: Why is this here? That [PR](https://github.com/kedro-org/kedro-viz/pull/1508) was never merged, right?

Reviewer 2: +1
